db2fce.control

```diff
@@ -2,4 +2,3 @@
 comment = 'DB2 compatibility environment for PostgreSQL'
 default_version = '0.0.12'
 module_pathname = '$libdir/db2fce'
-relocatable = false
```

Comment:
In my testing, I had to explicitly set `relocatable = true` or else it would still be marked as `extrelocatable = f`:

```
postgres=# SELECT * FROM pg_extension WHERE extname = 'db2fce';
 extname | extowner | extnamespace | extrelocatable | extversion | extconfig | extcondition 
---------+----------+--------------+----------------+------------+-----------+--------------
 db2fce  |       10 |        26511 | f              | 0.0.12     |           | 
(1 Zeile)

postgres=# ALTER EXTENSION db2fce UPDATE TO '0.0.13';
ALTER EXTENSION
postgres=# SELECT * FROM pg_extension WHERE extname = 'db2fce';
 extname | extowner | extnamespace | extrelocatable | extversion | extconfig | extcondition 
---------+----------+--------------+----------------+------------+-----------+--------------
 db2fce  |       10 |        26511 | f              | 0.0.13     |           | 
(1 Zeile)
```